src/node_http2.cc

```diff
@@ -71,6 +71,11 @@ Http2Scope::Http2Scope(Http2Session* session) {
   }
   session->flags_ |= SESSION_STATE_HAS_SCOPE;
   session_ = session;
+
+  // Always keep the session object alive for at least as long as the
+  // this scope is active.
```

Comment:
extra "the" or "this".

Reply:
@apapirovski yup, thanks for catching! 😄